tests/generative/variable_strategies.py

```diff
@@ -224,6 +225,18 @@ def date_difference_in_months(type_, frame):
     def date_difference_in_days(type_, frame):
         return binary_operation(datetime.date, frame, Function.DateDifferenceInDays)
 
+    @st.composite
+    def case(draw, type_, frame):
+        # case takes a mapping argument which is a dict with a bool series as keys and
+        # a series of `type_` as values.  It returns a series of the same type as its
+        # mapping values
+        # It also takes a default, which can be None or a series of type_
+        mapping = st.dictionaries(
+            series(bool, frame), series(type_, frame), min_size=1, max_size=5
+        )
+        default = st.one_of(st.none(), series(type_, frame))
+        return Case(draw(mapping), draw(default))
```

Comment:
I think that the RHS's of the mapping and the default need to have the possibility of being values. (As per https://github.com/opensafely-core/databuilder/pull/959#discussion_r1085342664.)

Reply:
Oh yes, of course.  I'll do that